helpdesk-service/helpdesk-service-client/pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>migbase64</artifactId>
             <version>2.2</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
I don't think it's required once you have imported mockito-all as a dependency.

Reply:
ok